pmd-apex/src/main/resources/rulesets/apex/quickstart.xml

```diff
@@ -126,7 +126,16 @@
          <property name="cc_block_highlighting" value="false" />
       </properties>
    </rule>
-   <rule ref="category/apex/performance.xml/AvoidDmlStatementsInLoops" message="Avoid DML Statements inside loops">
+   <rule ref="category/apex/performance.xml/AvoidDmlStatementsInLoops" message="Avoid DML Statements inside loops" deprecated="true">
```

Comment:
Github is showing this diff in an odd way. It's the addition of the `depecated` attribute on line 129. And the addition of the new `OperationWithLimitsInLoop` XML.